test/utils/role/test_icon.ts

```
@@ -0,0 +1,67 @@
+
+  it("falls back to smart_toy when the icon contains non-letter chars", () => {
+    assert.equal(roleIcon(roles, "broken"), "smart_toy");
+  });
+
+  it("falls back to star when the role is unknown", () => {
+    assert.equal(roleIcon(roles, "no-such-role"), "star");
+  });
+
+  it("accepts only lowercase letters and underscores as valid icons", () => {
+    const r: Role[] = [
+      { id: "a", name: "", icon: "valid_name", prompt: "", availablePlugins: [] },
```

ESLint

[error] Replace `·id:·"a",·name:·"",·icon:·"valid_name",·prompt:·"",·availablePlugins:·[]` with `⏎········id:·"a",⏎········name:·"",⏎········icon:·"valid_name",⏎········prompt:·"",⏎········availablePlugins:·[],⏎·····`

(prettier/prettier)